README.md

```diff
@@ -1,9 +1,55 @@
-# Indexing node
+![Query Nodes for Joystream](./banner.svg)
```

Comment:
Again:
Did you make this? Looks great, and this repo should have one! I just might forward it to the designers to make sure it's perfectly in line with the other ones though...